### PR TITLE
customize header, footer and homepage

### PR DESCRIPTION
**Pull request by HuynhKhoa1601**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Problem description
https://github.com/dataquest-dev/dspace-angular/issues/188

### CODE REVIEW

**Comment by milanmajchrak** [comment]
I think `:hover` background color in the menu should be the same like in the https://www.tul.cz/ . 
**TUL:**
image
**Your:**
image

**Comment by milanmajchrak** [comment]
Log in: the buttons doesn't change the color on `:hover` event. I think it should get a little bit darker like in the Vanilla.
image

I think you can copy `:hover` style from the tul.cz buttons: 
image

**Comment by milanmajchrak** [comment]
dspace.tul.cz has another info in the footer than Vanilla DSpace. Please update the footer and add there `Contact Us` option with redirection to `/contact`.

File: src/app/navbar/navbar.component.scss [hunk 1]
```diff
@@ -1,38 +1,38 @@
 nav.navbar {
-    border-bottom: 1px var(--ds-header-navbar-border-bottom-color) solid;
-    align-items: baseline;
+  border-bottom: 1px var(--ds-header-navbar-border-bottom-color) solid;
+  align-items: baseline;
+  background-color: var(--lt-tul-purple);
 }
```
Comment (milanmajchrak) [inline src/app/navbar/navbar.component.scss:6]: add here `:hover` colors

File: src/styles/_bootstrap_variables_mapping.scss [hunk 2]
```diff
@@ -93,7 +93,7 @@
   // Links
   //
   // Style anchor elements.
-  --bs-link-color: #{$link-color};
+  --bs-link-color: white;
```
Comment (milanmajchrak) [inline src/styles/_bootstrap_variables_mapping.scss]: I think better solution could be updating `link-color` what do you think?

File: src/themes/dspace/app/navbar/navbar.component.scss [hunk 3]
```diff
@@ -1,15 +1,19 @@
+@font-face {
+  font-family: 'TUL Mono';
```
Comment (milanmajchrak) [inline src/themes/dspace/app/navbar/navbar.component.scss:2]: tul.cz has in the navbar `0.85rem 'TUL Mono', sans-serif;` is it the same?

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

File: src/app/navbar/navbar.component.scss (hunk 1 above)
Comment (HuynhKhoa1601) [reply src/app/navbar/navbar.component.scss:6]: thanks

File: src/styles/_bootstrap_variables_mapping.scss (hunk 2 above)
Comment (HuynhKhoa1601) [reply src/styles/_bootstrap_variables_mapping.scss]: Agree

File: src/themes/dspace/app/navbar/navbar.component.scss (hunk 3 above)
Comment (HuynhKhoa1601) [reply src/themes/dspace/app/navbar/navbar.component.scss:2]: I'll add it

File: src/styles/_bootstrap_variables.scss [hunk 4]
```diff
@@ -29,17 +30,17 @@ $red:    #CF4444 !default;
...
 $theme-colors: (
-        primary: $blue,
-        secondary: $gray-700,
-        success: $green,
-        info: $cyan,
-        warning: $yellow,
-        danger: $red,
-        light: $gray-100,
-        dark: $dark
+        "primary": $blue,
```
Comment (milanmajchrak) [inline src/styles/_bootstrap_variables.scss]: why did you wrap it in the quotation marks? Are we sure it won't break other components which are using this `$theme-colors`?

**Comment by milanmajchrak** [comment]
image
On :hover the background color of TUL icon is not changed like in the tul.cz
image

**Comment by milanmajchrak** [comment]
image
Search button and Language Switch button background colors are not changed on the :hover event like in the rest of the menu.
It should be changed like in the e.g., Statistics menu button
image

**Comment by milanmajchrak** [comment]
You can see the border of the button on the :hover event: 
image

it should be smooth. Do you see it?

**Comment by milanmajchrak** [comment]
After :hover the hover-border of the button still could be seen: 
image

**Comment by milanmajchrak** [comment]
:hover colors are not the same in the menu options: 
image
image

File: src/styles/_bootstrap_variables.scss (hunk 4 above)
Comment (HuynhKhoa1601) [reply src/styles/_bootstrap_variables.scss]: oh yes, my bad. I'll revert it

Comment (HuynhKhoa1601) [reply src/styles/_bootstrap_variables.scss]: I just want to make it consistent

**Comment by milanmajchrak** [comment]
@HuynhKhoa1601  I think it looks good :) Thanks. 
I see one more fix and I believe for now the home page will be done.

My last suggestion is:
the dropdown menu is rolled down before the mouse cursor is changed. If you go through `All of DSpace` menu option slowly you should see this behaviour. 
image

**Review by milanmajchrak — APPROVED** [review]

**Pull request merged**